src/legacy/core_plugins/kbn_vislib_vis_types/public/components/options/metrics_axes/index.test.tsx

```diff
@@ -223,48 +226,10 @@ describe('MetricsAxisOptions component', () => {
       };
       comp.setProps({
         aggs: createAggs([agg]),
-        aggsLabels: agg.makeLabel(),
       });
 
       expect(setValue).not.toHaveBeenCalledWith(VALUE_AXES);
     });
-
-    it('should overwrite the custom title when the agg type changes', () => {
```

Comment:
Why were these tests removed?

Reply:
There were an effect which run `updateAxisTitle` on `aggsLabels` prop updates. The `aggsLabels` prop has been removed, and now there are the only effect which updates the title when the agg changes. (previously each agg object was mutated, but now it's a new obj each time, so to test the every param inside the agg was changed is excessive I think)